src/qt/qvalidatedlineedit.cpp

```diff
@@ -121,3 +131,45 @@ bool QValidatedLineEdit::isValid()
 
     return valid;
 }
+
+void QValidatedLineEdit::locateErrors()
+{
+    if (!errorLocator) return;
+
+    // Perform error location on the input
+    QString input = text();
+    std::vector<int> error_locations{};
+    std::string error_message;
+    errorLocator->locateErrors(input, error_message, &error_locations);
+
+    if (error_message.empty()) {
+        QMessageBox::information(this, "Error locator", "No errors detected.", QMessageBox::Ok);
+    } else {
+        QString error_text = "Error: " + QString::fromStdString(error_message) + "<br />";
+        error_text += "<pre>";
+        if (!error_locations.empty()) {
+            int prev_loc = 0;
+            for (int loc : error_locations) {
+                error_text += input.mid(prev_loc, loc - prev_loc);
+                error_text +=  "<FONT COLOR='#ff0000'>" + input[loc] + "</FONT>";
```

Comment:
[is something like this possible here?](https://github.com/bitcoin/bitcoin/blob/e3de7cb9039770e0fd5b8bb8a5cba35c87ae8f00/src/qt/rpcconsole.cpp#L877-L883)

Reply:
Sure, done

Reply:
I mean calling red what is red